web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -98,11 +101,11 @@
     on:mouseenter={onMouseEnter}
     on:mouseleave={onMouseLeave}
     tabindex={0}
-    on:click={isMultiSelectState ? onIconClickedHandler : undefined}
+    on:click={isMultiSelectState ? onIconClickedHandler : isStackSlideshow ? thumbnailClickedHandler : undefined}
     use:shortcut={{ shortcut: { key: 'Enter' }, onShortcut: thumbnailClickedHandler }}
   >
     {#if intersecting}
-      <div class="absolute z-20 h-full w-full {className}">
+      <div class="absolute z-20 h-[{height}px] w-[{width}px] {className}">
```

Comment:
Why do we need to change this?

Reply:
since we changed this from a <div> with `display: block` to an <a> with `display: flex`, leaving h-full and w-full here will overflow the actual image itself
<img width="229" alt="Screenshot 2024-06-07 at 4 21 55 PM" src="https://github.com/immich-app/immich/assets/31512688/e203e596-de64-43f0-8ea1-7d3bada3908f">

Reply:
Not sure this'll work because height and width aren't integers, so the classes will become something like `h-[281.298374982px]`, which isn't something tailwind supports. Plus, these classes are only for the wrapper surrounding the selecting button element. I wasn't able to reproduce the overflow with `h-full, w-full`. Where do you see it?

Reply:
do you have the most recent changes pulled? not sure why you aren't able to reproduce it, because I went back, edited only that one line and got this:

**h-full w-full**
<img width="228" alt="Screenshot 2024-06-07 at 6 13 55 PM" src="https://github.com/immich-app/immich/assets/31512688/c76e9802-d284-486d-9790-0267878fe176">

**h-[{height}px] w-[{width}px]**
<img width="229" alt="Screenshot 2024-06-07 at 6 14 12 PM" src="https://github.com/immich-app/immich/assets/31512688/5cebc9a7-47c0-4e0e-8842-03346218acb0">

Reply:
I do, I edited my comment to state I didn't realize this was only for the stack slideshow.
But the problem remains with the CSS class names when not in the context of a stack slideshow when using variables.

Reply:
unfortunately looks like h-auto w-auto resizes the white outline to be so much smaller that it only appears in the top left of the image

<img width="222" alt="Screenshot 2024-06-07 at 6 23 36 PM" src="https://github.com/immich-app/immich/assets/31512688/e3bfe66d-504d-4bcd-be5e-5ccc1bd4ed87">

Reply:
ok I just replaced `h-[{height}px] w-[{width}px]` with `style:width="{width}px" style:height="{height}px"` which works as expected

<img width="246" alt="Screenshot 2024-06-07 at 6 35 06 PM" src="https://github.com/immich-app/immich/assets/31512688/2c1c1997-f8e9-46ee-a5ab-47f828450bef">